populus/chain.py

```diff
@@ -714,7 +714,10 @@ def __enter__(self, *args, **kwargs):
         self.stack.enter_context(self.geth)
 
         if self.geth.is_mining:
-            self.geth.wait_for_dag(600)
+            # On Amazon EC2 2 VCPU small unit you get 32% of DAG generated
+            # within 600 seconds. Estimate is
+            # we can generate whole DAG in 40 minutes.
+            self.geth.wait_for_dag(60*40)
```

Comment:
Can we take an alternate approach to this and have this value configurable with an environment variable.  

Something akin to what is going on here https://github.com/pipermerriam/populus/blob/bc3052fe12a4916533b2270afd193761901f00f2/tests/cli/test_show_chain_sync_progress_helper_fn.py#L16